### PR TITLE
R2.15 documentation updates

### PR DESCRIPTION
**Pull request by inc1t3Ful**

### CODE REVIEW

File: website/docs/guides/release-notes.html.markdown [hunk 1]
```diff
@@ -12,11 +12,40 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
...
 ---
 
+## 2.15.0
+### Notes:
+- Supported Controller version: **UserConnect-6.0**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. New data sources:
+  - **data_source_aviatrix_firewall**
```
Comment (CyrusJavan) [inline website/docs/guides/release-notes.html.markdown]: Should these just be 
```
aviatrix_firewall
aviatrix_vpc_tracker
```

?

File: website/docs/guides/release-notes.html.markdown [hunk 2]
```diff
@@ -12,11 +12,40 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
...
+2. Implemented support for the option to manage attachment on either **aviatrix_vpn_profile** or **aviatrix_vpn_user** using ``manage_user_attachment`` (and ``profiles`` for the user)
+3. Implemented support for ``action`` under domain_names filters for **aviatrix_fqdn**
+4. Implemented support for adding VPN users under GeoVPN workflow
+5. Implemented support for specifying ``ha_peering_subnet`` for GCP **aviatrix_gateway**
+6. Implemented support for specifying ``ha_subnet`` for GCP **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+7. Implemented support for ``enable_ikev2`` for **aviatrix_site2cloud**
+
+### Enhancements:
+1. Updated **aviatrix_site2cloud**'s``tunnel_type`` to support "policy" and "route"-based options in Controller 6.0
+2. Added ``route_tables`` and ``route_tables_filter`` in **data_source_vpc**
```
Comment (CyrusJavan) [inline website/docs/guides/release-notes.html.markdown]:
```
aviatrix_vpc
```

Or 

```
data_source_aviatrix_vpc
```

?

Comment (inc1t3Ful) [reply website/docs/guides/release-notes.html.markdown]: Good point. Didnt originally catch that in Tim's changelog

File: website/docs/guides/release-notes.html.markdown (hunk 1 above)
Comment (inc1t3Ful) [reply website/docs/guides/release-notes.html.markdown]: I will update that and double check for existing Release Notes that use that incorrect syntax

Comment (zhiqiangjin) [reply website/docs/guides/release-notes.html.markdown]: No. Data sources and resources are different.

File: website/docs/guides/release-notes.html.markdown (hunk 2 above)
Comment (zhiqiangjin) [reply website/docs/guides/release-notes.html.markdown]: Same.

**Review by zhiqiangjin — APPROVED** [review]

**Pull request merged**